### PR TITLE
Update all the README files

### PR DESCRIPTION
**Pull request by mgiuca**
Update the README files to describe using the public versions of the demos.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,14 +18,34 @@ and outlines an API that we think solves this problem. But this is less about
...
 * For a detailed overview, see [Ballista Explained](docs/explainer.md).
-* In the `polyfill` directory, there is a partial polyfill. This doesn't allow
-  websites to register as handlers, but is sufficient to set up a bidirectional
-  action flow between two participating sites.
-* In the `handler` and `requester` directories are sample apps that work
-  together.
+* In the [`polyfill`](polyfill) directory, there is a polyfill that you can use
+  to write a requester that can fire actions at any handler, or a handler that
+  can receive actions from any requester.
+* In the [`handler`](handler) and [`requester`](requester) directories contain
```
Comment (sammc) [inline README.md]: "In the ... contain" doesn't make sense.

File: README.md [hunk 2]
```diff
@@ -18,14 +18,34 @@ and outlines an API that we think solves this problem. But this is less about
 proposing an API, and more about starting a conversation, so take a look, and
 let us know what you think (contact details below).
 
+## Demo
+
+We have a prototype that works in Chrome and Firefox. Try this:
+
+1. Go to
+   [handler-dot-chromium-ballista.appspot.com](https://handler-dot-chromium-ballista.appspot.com)
+   (Ballista Editor Demo), and register as an action handler.
```
Comment (sammc) [inline README.md]: This reads strangely. The person isn't registering as an action handler, they're allowing the app to register.

File: handler/README.md [hunk 3]
```diff
@@ -1,10 +1,11 @@
-# Ballista sample handler
+# Ballista Editor Demo
 
 **Author:** Matt Giuca <<mgiuca@chromium.org>>
```
Comment (sammc) [inline handler/README.md:4]: Should there be a live at ... link here too?

File: README.md (hunk 1 above)
Comment (mgiuca) [inline README.md]: Done.

File: README.md (hunk 2 above)
Comment (mgiuca) [inline README.md]: Done.

File: handler/README.md (hunk 3 above)
Comment (mgiuca) [inline handler/README.md:4]: Done.

**Comment by mgiuca** [comment]
PTAL (rebased).

**Comment by sammc** [comment]
LGTM

**Pull request merged**